cmd/background-controller/main.go

```diff
@@ -148,6 +148,7 @@ func main() {
 	eventGenerator := event.NewEventGenerator(
 		setup.EventsClient,
 		logging.WithName("EventGenerator"),
+		maxQueuedEvents,
```

Comment:
Can you check if the documentation reflect this latest status?

Reply:
Yes, this flag is documented on the website.